orttraining/orttraining/test/python/orttraining_test_ortmodule_mnist_deepspeed_config.json

```diff
@@ -0,0 +1,27 @@
+{
+    "train_batch_size": 32,
+    "steps_per_print": 2000,
+    "optimizer": {
+      "type": "SGD",
+      "params": {
+        "lr": 0.001
+      }
+    },
+    "scheduler": {
+      "type": "WarmupLR",
+      "params": {
+        "warmup_min_lr": 0,
+        "warmup_max_lr": 0.001,
+        "warmup_num_steps": 1000
+      }
+    },
+    "wall_clock_breakdown": true,
+    "fp16": {
+      "enabled": false
+    }
```

Comment:
missing comma here?
```suggestion
    },
```

Reply:
Thanks, this seems to be a stale broken copy of the file... I just pushed again with the fixes.